strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/testing/storage/repository/TestRepositoryContext.java

```diff
@@ -217,14 +218,8 @@ private Storage createStorage()
     {
         MutableStorage newStorage = new MutableStorage(testRepository.storageId());
         configurationManagementService.addStorageIfNotExists(newStorage);
-        try
-        {
-            storageManagementService.saveStorage(newStorage);
-        }
-        catch (IOException e)
-        {
-            throw new UndeclaredThrowableException(e);
-        }
+
+        ThrowingConsumer.unchecked(storageManagementService::saveStorage).accept(newStorage);
```

Comment:
same as above, we should add `throws IOException` here and use `ThrowingSupplier` where this method used